packages/strapi-design-system/src/DatePicker/DatePickerCalendar.js

```diff
@@ -91,6 +92,7 @@ export const DatePickerCalendar = ({
                         outsideMonth={outsideMonth}
                         onSelectDay={() => onChange(date)}
                         isSelected={isSelected}
+                        isCurrent={currentDate.getDate() === date.getDate()}
```

Comment:
This isn't enough because you are only comparing the day. Depending on when the current month started, you can highlight two days at the same time (ex: feb, 28 & march 28). 

![Screenshot 2022-03-28 at 12 42 47](https://user-images.githubusercontent.com/7756284/160382022-d4ffa68a-d10c-472e-8539-455345a18cfd.png)